clients/ops/admin-ui/src/features/datastore-connections/add-connection/manual/DSRCustomization.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, Center, Spinner, VStack } from "@fidesui/react";
+import { useAppSelector } from "app/hooks";
+import { useAlert, useAPIHelper } from "common/hooks";
+import { selectConnectionTypeState } from "connection-type/connection-type.slice";
+import {
+  useCreateAccessManualWebhookMutation,
+  useGetAccessManualHookQuery,
+  usePatchAccessManualWebhookMutation,
+} from "datastore-connections/datastore-connection.slice";
+import {
+  CreateAccessManualWebhookRequest,
+  CreateAccessManualWebhookResponse,
+  PatchAccessManualWebhookRequest,
+  PatchAccessManualWebhookResponse,
+} from "datastore-connections/types";
+import React, { useEffect, useRef, useState } from "react";
+
+import DSRCustomizationForm from "./DSRCustomizationForm";
+import { Field } from "./types";
+
+const DSRCustomization: React.FC = () => {
+  const mounted = useRef(false);
+  const { successAlert } = useAlert();
+  const { handleError } = useAPIHelper();
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [fields, setFields] = useState([] as Field[]);
+
+  const { connection } = useAppSelector(selectConnectionTypeState);
+
+  const { data, isFetching, isLoading, isSuccess } =
+    useGetAccessManualHookQuery(connection!.key);
+
+  const [createAccessManualWebhook] = useCreateAccessManualWebhookMutation();
+  const [patchAccessManualWebhook] = usePatchAccessManualWebhookMutation();
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const handleSubmit = async (values: Field[], _actions: any) => {
+    try {
+      setIsSubmitting(true);
+      const params:
+        | CreateAccessManualWebhookRequest
+        | PatchAccessManualWebhookRequest = {
+        connection_key: connection?.key as string,
+        // body: {
+        //   fields: values,
+        // },
+        body: { ...values } as any,
+      };
+      let payload:
+        | CreateAccessManualWebhookResponse
+        | PatchAccessManualWebhookResponse;
+      if (fields.length > 0) {
+        payload = await patchAccessManualWebhook(params).unwrap();
+      } else {
+        payload = await createAccessManualWebhook(params).unwrap();
+      }
+      setFields(payload.fields);
+      successAlert(
+        `DSR customization ${fields.length > 0 ? "updated" : "added"}!`
+      );
+    } catch (error) {
+      handleError(error);
+    } finally {
+      setIsSubmitting(false);
+    }
+  };
+
+  useEffect(() => {
+    mounted.current = true;
+    if (isSuccess && data) {
+      setFields(data.fields);
+    }
+    return () => {
+      mounted.current = false;
+    };
+  }, [data, isSuccess]);
+
+  return (
+    <VStack align="stretch" gap="24px">
+      <Box color="gray.700" fontSize="14px" w="572px">
+        Below you can select the required PII for this manually setup
+        Integration and then choose to remap a custom label for the DSR package
+        your users will download etc.
```

Comment:
Explanation/casing is confusing here - I guess manually is the adverb here and setup is supposed to be read like a verb, not a noun, I just had to read it a couple of times. Integration capitalized.  Is "etc" needed at the end?

Reply:
This verbiage was taken directly from the [Figma](https://www.figma.com/file/MTbZohIGvB4bq2f2wkiDHO/Fidesops-Management-UI?node-id=3458%3A49417
![Screen Shot 2022-09-14 at 2 21 56 PM](https://user-images.githubusercontent.com/68459950/190232794-25ba2b30-ea21-4956-a689-04b149fc0ed9.png)
)

Reply:
Let me follow up with Simon and Adam on the above verbiage.

Reply:
Here is the new verbiage from Simon:

`Customize your PII fields to create a friendly label name for your data subject request packages. This “DSR Package Label” is the label your data subject will see in their downloaded DSR package.`